samples/LocalEnvSecretStoreSample/LocalEnvSecretStoreSample.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>

```

Comment:
Nit: while we ship both `net6.0` and `net7.0` bits in the NuGet package, as .NET 6.0 is the LTS release, we should target it in our samples.